base/BUILD.gn

```diff
@@ -3513,6 +3512,13 @@ test("base_unittests") {
     "vlog_unittest.cc",
   ]
 
+  if (use_cobalt_customizations) {
+    sources += [
+      # Cobalt builds don't include build timestamps.
+      # "build_time_unittest.cc",
```

Comment:
Sorry for the confusion, I meant to leave it in the original list of sources and to add a `sources -=` here to be clear about which files we don't compile